<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Find</class>
 <widget class="QDialog" name="Find">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0,0">
         <property name="sizeConstraint">
          <enum>QLayout::SetNoConstraint</enum>
         </property>
         <item>
          <widget class="QRadioButton" name="radioButton">
           <property name="text">
            <string>By Group:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>â€”&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_2">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="currentText">
            <string/>
           </property>
           <property name="currentIndex">
            <number>-1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_2">
           <property name="text">
            <string>By Keyword:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Find</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Find</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>516</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>Find</receiver>
   <slot>set_project()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>534</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Find</receiver>
   <slot>query()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>597</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>574</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>activated(QString)</signal>
   <receiver>Find</receiver>
   <slot>by_group()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>58</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_2</sender>
   <signal>activated(QString)</signal>
   <receiver>Find</receiver>
   <slot>query()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>442</x>
     <y>2</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>by_category()</slot>
  <slot>by_tag()</slot>
  <slot>set_project()</slot>
  <slot>by_keyword()</slot>
  <slot>by_group()</slot>
  <slot>query()</slot>
 </slots>
</ui>
